apps/remix-ide/src/app/tabs/test-tab.js

```diff
@@ -162,7 +194,11 @@ module.exports = class TestTab extends ViewPlugin {
         const method = result.assertMethod === 'ok' ? '' : result.assertMethod
         const expected = result.assertMethod === 'ok' ? `'true'` : result.expected
         this.testsOutput.appendChild(yo`
-          <div class="bg-light mb-2 ${css.testFailure} ${css.testLog} d-flex flex-column text-danger border-0" id="UTContext${result.context}">
+          <div
+            class="bg-light mb-2 ${css.testFailure} ${css.testLog} d-flex flex-column text-danger border-0"
+            id="UTContext${result.context}"
+            onclick=${() => this.highlightLocation(result.location, this.rawFileName)}
```

Comment:
should this be `highlightLocation` or `jumpToLocation` ?

Reply:
from test-tab point of view it's more highlightL, form editors point of view it's jump.. but I would say both will work

Reply:
is it `highlighting` , i think it doesn't work for me